content/departments/product-engineering/engineering/cloud/devops/managed/index.md

```diff
@@ -102,6 +103,30 @@ The main limitation of this model is that an underlying GCP infrastructure outag
 
 Access can be requested in #it-tech-ops WITH manager approval.
 
+### Monitoring and alerting
+
+<span class="badge badge-note">SOC2/CI-86</span>
+
+Each managed instance is created in an isolated GCP project and is monitored and alerted via:
+
+- [uptime check](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/main/modules/terraform-managed-instance-new/infrastructure.tf#L553) configured in dedicated GCP managed instance project
+- [instance performance metric alerts](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/main/monitoring/alerting.tf) configured in scoped project for all managed instances
+
+Alerting flow:
+
+1. When alert is triggered, it is sent to Opsgenie channel:
+
+- [uptime check channel](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/main/modules/terraform-managed-instance-new/infrastructure.tf#L557)
+- [metrics monitoring channel](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/main/monitoring/alerting.tf#L24)
+
+2. From Opsgenie, alert is sent to [on-call DevOps](../index.md#on-call) and Slack channels (#opsgenie, #cloud-devops).
+
+3. On-call DevOps has to decide, what is the alert type and if [incident](../../../../engineering/process/incidents/index.md) should be opened and follow the procedure to perform the incident. On-call DevOps should use [managed instances operations](./operations.md) to check, assess and repair broken managed instance.
+
+4. When alert is closed via incident resolution, post-mortem actions has to be assigned and performed.
```

Comment:
How do we maintain post-mortems? Do we use Opsgenie for it or do we have a Google Drive directory for it?

Reply:
Thx @sanderginn ! I have added links to post-mortem and process for incidents in handbook - I would not copy-paste it here, but rather use the links if that's ok for you?

Reply:
:D I didn't open the incident link. That did describe everything I commented on. But you made it more explicit so 👍